### DISCUSSION TITLE
Error using library Keras in R

### DISCUSSION THREAD

**Discussion by andrewanggoh** [teamalgoritma/community#721] [Q&A] [answered]
Permisi, saya mengalami kendala error ketika ingin menggunakan library Keras pada RStudio seperti pada foto yang terlampir. Pesan errornya adalah sebagai berikut:

"Error in system(cmd, intern = intern, wait = wait | intern, show.output.on.console = wait, :
'/c' not found"

Mohon bantuannya Kak. Terima kasih sebelumnya🙏

![image](https://github.com/teamalgoritma/community/assets/127578298/31ec901e-e0a9-4682-a87c-c46ec47bcc4f)

**Comment by arkanayudhistira** [comment]
Dear @andrewanggoh,

Apakah error tersebut hanya muncul ketika menjalankan `library(keras)` saja atau ketika menjalankan code yang lain? Jika hanya muncul di library(keras) saja, mungkin dapat dicoba melakukan proses pembuatan virtual environment baru sesuai dengan proses pada handbook (pastikan versi tensorflow adalah 2.12 dengan melihat pada `pip list`).

Semoga membantu.

**Reply by andrewanggoh** [reply]
Dear @arkanayudhistira, 

Terima kasih untuk jawabannya Kak. Error tersebut sejauh ini hanya muncul ketika saya menjalankan library(keras) dan code yang terkait dengan library(keras). Untuk virtual environment dan langkah - langkah lain sudah saya coba lakukan sesuai dengan handbook, namun masih mengalami error yang sama.

**Reply by divakartika** [reply, marked as answer]
Dear @andrewanggoh 

Apakah terdapat nama folder dalam C:\Users\ yang menggunakan spasi?

**Reply by kevbow** [reply]
Halo @andrewanggoh, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
